File: easy/1018-binary-prefix-div-by-5/sol.cpp
#include <vector>

class Solution {
public:
    vector<bool> prefixesDivBy5(vector<int>& nums) {
        vector<bool> ans(nums.size(), false);
        int num = 0;

        for(int i=0;i<nums.size();i+=1) {
            num <<= 1;
            num = (num + nums[i]) % 10;
            ans[i] = num==0 || num==5;
        }

        return ans;
    }
};

File: medium/0334-increasing-triplet-subsequence/sol.py
class Solution:
    def increasingTriplet(self, nums: List[int]) -> bool:
        INF = float('inf')
        a,b = INF, INF
        ans = False

        for num in nums:
            if num < a:
                a = num
            elif num < b and num != a:
                b = num
            elif num > b:
                ans = True
                break
        
        return ans